src/front/wgsl/lexer.rs

```diff
@@ -1,4 +1,5 @@
 use super::{conv, Error, Token, TokenSpan};
+use std::ops::Range;
```

Comment:
this may be a little tricky given that `crate::arena::Range` is often imported in other modules